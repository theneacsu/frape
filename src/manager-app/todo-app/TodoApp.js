import React from 'react'
import TodoInputArea from './todo-input-area/TodoInputArea'
import TodoFilterArea from './todo-filter-area/TodoFilterArea'
import TodoList from './todo-list/TodoList'

const TodoApp = () => (
  <div>
    <TodoInputArea />
    <TodoFilterArea />
    <TodoList />
  </div>
)

export default TodoApp
