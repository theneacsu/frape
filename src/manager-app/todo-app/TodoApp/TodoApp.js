import React from 'react'
import TodoInputArea from '../components/todo-input-area/TodoInputArea'
import TodoFilterArea from '../components/todo-filter-area/TodoFilterArea'
import TodoList from '../components/todo-list/TodoList'
import styles from './TodoApp.module.css'

const TodoApp = () => (
  <div className={styles.div}>
    <div className="container">
      <TodoInputArea />
      <TodoFilterArea />
      <TodoList />
    </div>
  </div>
)

export default TodoApp
