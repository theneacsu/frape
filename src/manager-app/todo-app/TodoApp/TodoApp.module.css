:global(.input) {
  outline: none;
  border: none;
  padding: 1.5rem;
  margin-bottom: 0.4rem;
  color: #47667c;
  border-radius: 1rem;
}

:global(.input::placeholder) {
  color: #ccc;
}

:global(.container) {
  width: 90%;
  max-width: 500px;
  margin: 0 auto;
  display: flex;
  flex-direction: column;
}

.div {
  background-color: #f7ebc6;
  padding: 1.4rem 1.6rem;
  display: flex;
  flex: 1;
}
