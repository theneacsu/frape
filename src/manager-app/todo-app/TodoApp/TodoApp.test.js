import React from 'react'
import { shallow } from 'enzyme'
import TodoApp from './TodoApp'

test('Should render correctly the TodoApp component', () => {
  const wrapper = shallow(<TodoApp />)
  expect(wrapper).toMatchSnapshot()
})
