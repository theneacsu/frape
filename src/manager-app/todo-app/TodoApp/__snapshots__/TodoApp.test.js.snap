// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render correctly the TodoApp component 1`] = `
<div
  className="div"
>
  <div
    className="container"
  >
    <Connect(TodoInputArea) />
    <Connect(TodoFilterArea) />
    <Connect(TodoList) />
  </div>
</div>
`;
